python-scripts/connections/connections.py

```diff
@@ -0,0 +1,32 @@
+def connectDB(db): 
+    
+    '''This function is used to connect to the HU server. 
+    Please provide a valid database name in order to connect'''
+
+    import pymssql 
+    connect = pymssql.connect(
+        server='DBND25.medewerkers.ad.hvu.nl',
```

Comment:
Ik vraag me ff af in hoeverre het expliciet benoemen van onze servernaam veilig is, weet jij dat?

Reply:
ja goeie vraag kan in principe geen kwaad maar voor de zekerheid eruit laten?

Reply:
heb er een input veld van gemaakt